vivbin

```diff
@@ -43,7 +43,26 @@ def main():
     vw.verbose = args.verbose
 
     if args.option is not None:
-        vw.config.parseConfigOption(args.option)
+        if optarg in ('-h','?'):
+            print(vw.config.reprConfigPaths())
+            sys.exit(-1)
+
+        try:
+            vw.config.parseConfigOption(args.option)
+        except e_config.ConfigNoAssignment, e:
+            print(vw.config.reprConfigPaths())
+            print
```

Comment:
all new deltas must exclusively use "print as function" syntax to avoid digging ourselves in deeper 2.7 vs 3.x debt :)